yin_yang/helpers.py

```diff
@@ -0,0 +1,50 @@
+import subprocess
+
+
+"""Check output of a command.
+
+This is a helper method which will change how we check output depending on if
+The application is running in a Flatpak or not.
+"""
+
+
+def check_output(args, universal_newlines=False) -> bytes:
+    try:
+        output = subprocess.check_output(
+            args=args, universal_newlines=universal_newlines
+        )
+        return output
+    except FileNotFoundError:
+        flatpak_args = ["flatpak-spawn", "--host"] + args
+        return subprocess.check_output(
+            args=flatpak_args, universal_newlines=universal_newlines
+        )
+
+
+def check_call(command, stdout=None) -> int:
+    try:
+        return subprocess.check_call(command, stdout=stdout)
+    except FileNotFoundError:
+        flatpak_args = ["flatpak-spawn", "--host"] + command
+        return subprocess.check_call(flatpak_args, stdout=stdout)
+
+
+def is_flatpak() -> bool:
+    try:
+        subprocess.run('lookandfeeltool')
```

Comment:
Seems like there is an environment variable for that: `container`

src: https://stackoverflow.com/questions/75274925/is-there-a-way-to-find-out-if-i-am-running-inside-a-flatpak-appimage-or-another

Reply:
Or even better, we can look for a file `/.flatpak-info`

https://docs.flathub.org/docs/for-app-authors/requirements/#strongly-discouraged

Reply:
Great find, I didn't think to check the flathub docs. I updated the is_flatpak def to check for that file instead.